sdk/core/build.rs

```diff
@@ -0,0 +1,9 @@
+use rustc_version::version;
+
+fn main() {
+    let version = match version() {
+        Ok(version) => version.to_string(),
+        Err(_) => "unknown".to_string(),
```

Comment:
nit: `.to_owned()` preferred for reasons: https://users.rust-lang.org/t/to-string-vs-to-owned-for-string-literals/1441

Reply:
Thanks for the link. I actually had that originally thinking it would effectively move the string reference instead of reallocate, but wasn't sure it was right. I have been going to source for some of these methods, since the docs aren't always clear on implementation details like that.

Reply:
Actually, in that same article is a link to a [PR](https://github.com/rust-lang/rust/pull/32586) merged in 2016 that specializes `ToString` for `str` and does the same as `to_owned()`; but, just FYI. I will change it for optics, but seems it's no longer necessary.